src/BloomBrowserUI/bookEdit/js/origami.js

```diff
@@ -75,14 +77,16 @@ function layoutToggleClickHandler() {
 // Event handler to split the current box in half (vertically or horizontally)
 function splitClickHandler() {
     var myInner = $(this).closest('.split-pane-component-inner');
-    if ($(this).hasClass('splitter-top'))
+    if ($(this).hasClass('add-top'))
```

Comment:
I'm not convinced that 'add' is a better name than 'splitter'. It's true that you're not really splitting the pane, but adding another one beside it. But the visual effect is splitting, so that unambiguously tells me what the control that has this class does. On the other hand, 'add' could mean all kinds of other things...add a choice of image or text to a new pane, add some white space somewhere, maybe it's not related to the splitter at all but just has to do with controlling margins...maybe add-pane or add-panel? Or maybe I'm just missing something, I don't understand how all these classes work very well.